example/storage-provider-secrets/04-google-cloud-storage-secret.yaml

```diff
@@ -7,3 +7,4 @@ type: Opaque
 data:
   serviceaccount.json: ...
 #  storageAPIEndpoint: # http[s]://host[:port]/storage/v1/
+#  enableGCSEmulator: # True (case-insensitive)
```

Comment:
Why case-insensitive ?

Reply:
> Why case-insensitive ?

As it gives the user more flexibility of writing it in any form he/she chooses and not necessarily restricting them to just use "TRUE" only. They can use "True", "true", "TRUE", etc.. And similarly for the "faLsE" also 

> To improve clarity, we can use emulatorEnabled, which clearly indicates whether the emulator is enabled or not.

I think this is similar to the other comment, I stated in the [reply](https://github.com/gardener/etcd-backup-restore/pull/697#discussion_r1448708918) why we use that, since these two are inter linked, changing one requires changing others. I'm open to change, this can be discussed in the other comment [here](https://github.com/gardener/etcd-backup-restore/pull/697#discussion_r1448325839)

Reply:
For future context: The variable has been renamed to `emulatorEnabled` instead of GCS specific naming to make it uniform across multiple fake providers i.e Azure, etc.

Reply:
The codebase is split in naming convention for `bool` types: some variables use `XEnabled` and some variables use `EnableX`, where the latter is more common from what I could find. Would it not be clearer to name this common flag to `enableEmulator`? What do you think @seshachalam-yv @anveshreddy18?